conda/testing/gateways/helpers.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2012 Anaconda, Inc
+# SPDX-License-Identifier: BSD-3-Clause
+
+import json
+import os
+from pathlib import Path
+from subprocess import run
+
+try:
+    run(["minio", "-v"], check=True)
+    have_minio = True
```

Comment:
That reminds me, shouldn't this be named "has_minio"?

Reply:
I was following the `have_boto` flag convention:

https://github.com/conda/conda/blob/cd566fb97a5a0f6855fbdb1d0bea8500f6cdc0a8/conda/gateways/connection/adapters/s3.py#L13

But anyway we have a different way to flag this now (via MINIO_EXE), so not important!

Reply:
Ah, d'oh!